### ISSUE TITLE
Line numbers are not aligned on Chrome

### ISSUE THREAD

**Issue by siebenschlaefer** [exercism/website-copy#1534]
On Chrome (80.0.3987.163, Linux and Windows) the line numbers are not aligned with the lines of the source code.

![Screenshot from 2020-04-07 03-57-30](https://user-images.githubusercontent.com/1228739/78621916-03069280-7884-11ea-9244-9035f1c21744.png)

On Firefox (74.0.1, Linux and Windows) everything is fine.

![Screenshot from 2020-04-07 04-02-58](https://user-images.githubusercontent.com/1228739/78622175-bd969500-7884-11ea-981c-9cfb9d72b50e.png)

Screenshots from https://exercism.io/mentor/solutions/b7456d081946490692f86997f887650f

**Comment by corneliusroemer** [comment]
@ihid Misclassified issue: Belongs to main repo https://github.com/exercism/exercism

**Comment by siebenschlaefer** [comment]
@corneliusroemer Are you sure? This is a display issue on the website.

**Comment by iago-lito** [comment]
Duplicate of https://github.com/exercism/exercism/issues/4350 ?

**Comment by corneliusroemer** [comment]
@siebenschlaefer Yes, it belongs to the main repo issue tracker, see https://github.com/exercism/exercism/issues/5433
The situation is confusing, it confused me too. That's why I opened the referenced issue and labelled issues to be moved.
